README.md

```diff
@@ -1,2 +1,88 @@
-# yotas
-Oss's monetisation system
+# Yotas
+
+Oss's reward system
+
+[French](#french)
+
+[English](#english)
+
+# english
+
+Yotas is OSS's reward system, we would like our open source contributors across the organisation to be rewarded for the effort and energy they put into the platform. Each interaction they have on GitHub will give them a certain number of yotas which they can exchange for goodies in an OSS shop, mentoring, ebooks, etc....
```

Comment:
This is awesome, please some minor corrections

Yotas is an open source engagement awarding system. Open source contributors across the organisation are awarded for the effort and energy they put into the platform. Each interaction they have on GitHub will give them a certain number of Yotas which can be traded for goodies, mentoring, ebooks (etc...) in an OSS shop.

Reply:
done